docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -257,20 +257,31 @@ policies have the prefix ``hip_``.
                                                          Note that the thread-block
                                                          size and grid size must be
                                                          provided, there is no default.
- cuda/hip_exec_occ_calc<BLOCK_SIZE>        forall        Execute loop iterations
+ cuda/hip_exec_occ_max<BLOCK_SIZE>         forall        Execute loop iterations
                                                          mapped to global threads via
                                                          grid striding with multiple
                                                          iterations per global thread
                                                          in a GPU kernel launched
                                                          with given thread-block
                                                          size and grid size bounded
                                                          by the maximum occupancy of
-                                                         the kernel. Note that the
-                                                         thread-block size must
-                                                         be provided, there is no
-                                                         default. Note this can improve
-                                                         reducer performance in kernels
-                                                         with large iteration counts.
+                                                         the kernel.
+ cuda/hip_exec_occ_calc<BLOCK_SIZE>        forall        Similar to the occ_max
+                                                         policy but may use less
+                                                         than the maximum occupancy
+                                                         of the kernel for performance
+                                                         reasons.
+ cuda/hip_exec_occ_fraction<BLOCK_SIZE,    forall        Similar to the occ_max
+     RAJA::Fraction<size_t,                              policy but use a fraction
+        numerator, denominator>>                         of the maximum occupancy
+                                                         of the kernel.
+ cuda/hip_exec_occ_custom<BLOCK_SIZE,      forall        Similar to the occ_max
+     Concretizer>                                        policy but the grid size
+                                                         is determined by the
+                                                         concretizer.
+ cuda/hip_exec_rec_for_reduce<BLOCK_SIZE>  forall        The cuda/hip exec policy
+                                                         that is recommended for
+                                                         use with reducers.
```

Comment:
Is it important or helpful to say why it is recommended?

Reply:
I added a bit more explanation